extensions/ui/start.cmd.tsx

```diff
@@ -23,18 +40,85 @@ export class StartCmd implements Command {
      */
     private ui: UiMain,
 
-    private logger: Logger
-  ) {}
+    private logger: Logger,
+
+    private pubsub: PubsubMain
+  ) {
+    pubsub.sub('teambit.bit/ui', this.eventsListeners);
+    pubsub.sub('teambit.bit/webpack', this.eventsListeners);
+    pubsub.sub('teambit.bit/bundler', this.eventsListeners);
+  }
+
+  private eventsListeners = (event) => {
+    // console.log('--event-->: ', event)
+
+    switch (event.type) {
+      case 'components-server-started':
```

Comment:
we should think of a way to share the event types not like this. someone will change the name in the other side and it will be break, no one will know about it.

Reply:
wip